Deployment/BootMenuNoKVM.ps1

```diff
@@ -1,71 +0,0 @@
-﻿# Copyright (c) Microsoft Corporation. All rights reserved.
-# See LICENSE.txt in the project root for license information. 
-
-<# 
- 
```

Comment:
I am getting an error when I instantiate the script to prep the cloud.vhdx

>>
The property 'Name' cannot be found on this object. Verify that the property exists and can be set.
At C:\Azure_Installer\asdk-installer.ps1:1281 char:1
+ $Runspace_Jobs_Properties.Name = "Jobs"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : PropertyAssignmentException

Initialize environment. Please wait...
>>

any thoughts?  Windows 2012R2 system